### ISSUE TITLE
Reply from LUA

### ISSUE THREAD

**Issue by Brawl345** [vysheng/tg#791]
There is currently no way to reply to a message in a lua script :/
Can you add this please? :)

**Comment by LukeLR** [comment]
You are able to send messages, that's all you need. Neither possible as well as needed to add a reply function.

**Comment by Brawl345** [comment]
No, thats not "all I need". So, why do you know that this is not needed? I don't think you can speak for everyone...

**Comment by LukeLR** [comment]
Replying to your message is nothing else as sending a message to the sender's ID. Or do you need the sender's message to appear again?

**Comment by Brawl345** [comment]
Yes, I need the sender's message to appear again in the message.

**Comment by nullifye** [comment]
i'd updated lua-tg.c to support message reply.
i hv made a pull request https://github.com/vysheng/tg/pull/800

**Comment by mi4ka** [comment]
@nullifye Great work!
What about methods like `reply_photo`, `reply_audio` and so on?
